Gems/Atom/Feature/Common/Code/Source/DiffuseGlobalIllumination/DiffuseCompositePass.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <DiffuseGlobalIllumination/DiffuseCompositePass.h>
+#include <DiffuseGlobalIllumination/DiffuseProbeGridFeatureProcessor.h>
+
+namespace AZ
+{
+    namespace Render
+    {
+        // --- Dedicated class for disabling ---
+
+        RPI::Ptr<DiffuseCompositePass> DiffuseCompositePass::Create(const RPI::PassDescriptor& descriptor)
+        {
+            RPI::Ptr<DiffuseCompositePass> pass = aznew DiffuseCompositePass(descriptor);
+            return AZStd::move(pass);
```

Comment:
This is minor, but we use this pattern a lot and I don't quite see why we can't use do 

return aznew DiffuseCompositePass();

and let NRVO do its thing.

Reply:
This pattern was taken directly from a pass in the development branch. I forget which one now since it has been so long but I can try and find it. Might be the SSAO pass? That looks the same at a glance. I can change it if it poses a problem though. Can you point to a pass that conforms to the standard you are looking for that I can use as a guide, or is it literally only the return line that needs to change?